profile/src/main/kotlin/com/imashnake/animite/profile/tabs/Favourites.kt

```diff
@@ -0,0 +1,72 @@
+package com.imashnake.animite.profile.tabs
+
+import android.util.Log
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.util.fastForEach
+import com.imashnake.animite.api.anilist.sanitize.media.Media
+import com.imashnake.animite.api.anilist.sanitize.profile.User.MediaCollection.NamedList
+import com.imashnake.animite.core.ui.LocalPaddings
+import com.imashnake.animite.core.ui.MediaSmall
+import com.imashnake.animite.core.ui.MediaSmallRow
+import com.imashnake.animite.media.R as mediaR
+import com.imashnake.animite.core.R as coreR
+
+@Composable
+fun FavouritesTab(
+    favouriteLists: List<NamedList>,
+    modifier: Modifier = Modifier,
+) {
+    val scrollState = rememberScrollState()
+
+    Column(
```

Comment:
A users favorites list could be _massive_, this should probably be lazy

Reply:
Each user favourite list could be massive and `MediaSmallRow` is already a `LazyRow` 🤔